packages/react-bindings/src/hooks/index.ts

```diff
@@ -1,5 +1,12 @@
-export * from './call';
-export * from './participants';
+import * as CallStateHooks from './callStateHooks';
+
 export * from './permissions';
 export * from './store';
-export * from './user';
+// export * from './callStateHooks';
```

Comment:
It seems that it can be removed

Reply:
Kept this export, in order to keep this PR backwards compatible. I'll remove this export once together with the removal of `useCallMetadata`.